app/components/AccessTypeCell.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+import type { CellContext } from '@tanstack/react-table'
+
+import type { IdentityType } from '@oxide/api'
+
+/**
+ * Display whether this row displays a User or Group.
+ */
+export const AccessTypeCell = <RowData extends { identityType: IdentityType }>(
+  info: CellContext<RowData, string>
+) => {
+  const identityType = info.row.original.identityType
+  return <span>{identityType === 'silo_group' ? 'Group' : 'User'}</span>
+}
```

Comment:
Now that this cell is dedicated to `identityType` and used in a column where the accessor is specified as `identityType`, you don't have to do shenanigans to get that value — it's now in `info.getValue()`. It can also be a fragment instead of a span.

```suggestion
export const AccessTypeCell = <RowData extends { identityType: IdentityType }>(
  info: CellContext<RowData, IdentityType>
) => {
  return <>{info.getValue() === 'silo_group' ? 'Group' : 'User'}</>
}
```

Another approach, driven by my desire to get rid of all the noise in the type signature, would be to decouple this from React Table altogether and do something like this:

```ts
export const accessTypeLabel = (identityType: IdentityType) =>
  identityType === 'silo_group' ? 'Group' : 'User'
```

```ts
colHelper.accessor('identityType', {
  header: 'Type',
  cell: (props) => accessTypeLabel(props.getValue()),
}),
```

That's how they tend to do it in the RT docs: 

https://tanstack.com/table/latest/docs/guide/column-defs#cell-formatting

We in the past have tended toward these `Cell` components that let you pass them directly as `cell: WhateverCell` because it looks cleaner at the call site, but I think the complexity of this refactor shows the downside.